src/FSharp.Compiler.Interactive.Settings/fsiaux.fs

```diff
@@ -81,59 +108,95 @@ type InteractiveSession()  =
-      addedPrinters <- Choice1Of2 (typeof<'T>, (fun (x:obj) -> printer (unbox x))) :: addedPrinters
-
-    member self.EventLoop
-       with get () = evLoop
-       and set (x:IEventLoop)  = evLoop.ScheduleRestart(); evLoop <- x
-
-    member self.AddPrintTransformer(printer : 'T -> obj) =
-      addedPrinters <- Choice2Of2 (typeof<'T>, (fun (x:obj) -> printer (unbox x))) :: addedPrinters
-
-    member internal self.SetEventLoop (run: (unit -> bool), invoke: ((unit -> obj) -> obj), restart: (unit -> unit)) =
-      evLoop.ScheduleRestart()
-      evLoop <- { new IEventLoop with 
-                     member _.Run() = run()
-                     member _.Invoke(f) = invoke((fun () -> f() |> box)) |> unbox
-                     member _.ScheduleRestart() = restart() }
-    
-[<assembly: CodeAnalysis.SuppressMessage("Microsoft.Design", "CA1009:DeclareEventHandlersCorrectly", Scope="member", Target="FSharp.Compiler.Interactive.InteractiveSession.#ThreadException")>]
-do()
-  
-  
-module Settings = 
+    member _.FloatingPointFormat
+        with get () = fpfmt
+        and set v = fpfmt <- v
+
+    member _.FormatProvider
+        with get () = fp
+        and set v = fp <- v
+
+    member _.PrintWidth
+        with get () = printWidth
+        and set v = printWidth <- v
+
+    member _.PrintDepth
+        with get () = printDepth
+        and set v = printDepth <- v
+
+    member _.PrintLength
+        with get () = printLength
+        and set v = printLength <- v
+
+    member _.PrintSize
+        with get () = printSize
+        and set v = printSize <- v
+
+    member _.ShowDeclarationValues
+        with get () = showDeclarationValues
+        and set v = showDeclarationValues <- v
+
+    member _.ShowProperties
+        with get () = showProperties
+        and set v = showProperties <- v
+
+    member _.ShowIEnumerable
+        with get () = showIEnumerable
+        and set v = showIEnumerable <- v
+
+    member _.ShowIDictionary
+        with get () = showIDictionary
+        and set v = showIDictionary <- v
+
+    member _.AddedPrinters
+        with get () = addedPrinters
+        and set v = addedPrinters <- v
+
+    member _.CommandLineArgs
+        with get () = args
+        and set v = args <- v
+
+    member _.AddPrinter(printer: 'T -> string) =
+        addedPrinters <-
+            Choice1Of2(typeof<'T>, (fun (x: obj) -> printer (unbox x)))
+            :: addedPrinters
+
+    member _.EventLoop
+        with get () = evLoop
+        and set (x: IEventLoop) =
+            evLoop.ScheduleRestart()
+            evLoop <- x
+
+    member _.AddPrintTransformer(printer: 'T -> obj) =
+        addedPrinters <-
+            Choice2Of2(typeof<'T>, (fun (x: obj) -> printer (unbox x)))
+            :: addedPrinters
+
+    member internal self.SetEventLoop(run: (unit -> bool), invoke: ((unit -> obj) -> obj), restart: (unit -> unit)) =
+        evLoop.ScheduleRestart()
+
+        evLoop <-
```

Comment:
@dsyme I don't believe this file is being checked by the CI.
It would fail here as this was changed:
https://github.com/dotnet/fsharp/blob/217fc1fac7483c6cce830db85619403181cfb547/.editorconfig#L9

Reply:
Hmm: 
```
C:\GitHub\dsyme\fsharp>git checkout ft18
Switched to branch 'ft18'

C:\GitHub\dsyme\fsharp>dotnet fantomas . -r --check
No changes required.
```

Reply:
OK, fixed, thanks!